classes/ThemeScanner.php

```diff
@@ -33,11 +33,21 @@ public static function scan()
      */
     public function scanForMessages()
     {
+        $this->setMessagesAsNotFound();
```

Comment:
Inline this method as 
```suggestion
        // Set all messages as not found to be updated one by one.
        Message::query()->update(['found' => false]);
```